IIPS/iip-35.md

```diff
@@ -91,4 +91,4 @@ TBD
 
 ## Copyright
 
-Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```

Comment:
Please no not touch this last line that is not related to your changes. It should have newline at EOF.

Reply:
I checked.